src/main/index.js

```diff
@@ -116,3 +118,17 @@ ipcMain.on('get-app-details', event => {
         });
     }
 });
+
+/**
+ * Let's store the full path to the executable if nRFConnect was started from a built package.
+ * This execPath is stored in a known location, so e.g. VS Code extension can launch it even on
+ * Linux where there's no standard installation location.
+ */
+if (electronApp.isPackaged) {
```

Comment:
```suggestion
if (process.env.NODE_ENV === "production") {
```

We decided to use only `process.env.NODE_ENV` instead of `app.isPackaged` and the `electron-is-dev` package in order to keep these development/production build checks consistent.

Reply:
I guess that works too, though I am not totally convinced it checks the same thing and the check must work in the main process, not only in the renderer.